### ISSUE TITLE
Single quotes don't work?

### ISSUE THREAD

**Issue by kgrossjo** [dthree/cash#21]
Of the following two lines, the first line fails, the second line works:

```sh
alias ll='ls -la'
alias ll="ls -la"
```

I tried it twice, with the same results:

1. Running PowerShell inside ConEmu, then typing `cash`
2. Running `cmd.exe` inside its own window, then typing `cash`

The error I get is this:

```
$ alias ll='ls -la'

  Invalid option: 'l'. Showing Help:

  Usage: alias [options] [name...]


  Options:

    --help  output usage information
    -p  print all defined aliases in a reusable format
```

**Comment by dthree** [comment]
Okay traced this back to a bug in Vorpal's argument parsing. Thanks for pointing out.

**Comment by dthree** [comment]
Fixed in Vorpal `v1.10.2`, Cash `v0.4.0`.

https://github.com/dthree/vorpal/commit/c2036c9830b88bb2c74b762258122731e65f1017

**Comment by dthree** [comment]
Fixed in Vorpal `v1.10.2`, Cash `v0.4.0`.

https://github.com/dthree/vorpal/commit/c2036c9830b88bb2c74b762258122731e65f1017
